test/integration/basic/basic.go

```diff
@@ -179,5 +188,6 @@ func (r *BasicTestRunner) Run(ctx context.Context) {
 		r.T.Logf("Testing: %s\n", c.doc)
 		r.Setup(ctx, c.createOptsPublic, c.createOptsPrivate)
```

Comment:
"defer r.TearDown(ctx)" here instead of below

Reply:
Makes sense.